app/assets/config/manifest.json.erb

```diff
@@ -0,0 +1,25 @@
+{
+  "background_color": "white",
+  "description": "Break the awkward silence!",
+  "display": "fullscreen",
+  "name": "de.Bored",
+  "icons": [
+    <% files = Dir.entries(Rails.root.join("app/assets/images/icons/")).select {|f| !File.directory? f
+    } %>
+    <% files.each_with_index do |file, index| %>
+      <% match = file.match(/.+-(?<size>\d{
+      2,
+      3
+    }x\d{
+      2,
+      3
+    }).png/) %>
```

Comment:
Any reason why this regex is on multi-line? from the tutorial it just looks like this:
```erb
<% files.each_with_index do |file, index| %>
      <% match = file.match(/.+-(?<size>\d{2,3}x\d{2,3}).png/) %>
      {
        "src":
```

Reply:
whaaat I have no idea how it ended up like that. Will fix right away!

Reply:
Done :)